src/yb/client/client.cc

```diff
@@ -1065,6 +1065,13 @@ Status YBClient::GetYsqlCatalogMasterVersion(uint64_t *ysql_catalog_version) {
   return Status::OK();
 }
 
+Status YBClient::GetTServerUUID(std::string &node_uuid) {
```

Comment:
Generally, out params should be passed as a ptr, instead of a reference.

better yet, just return the uuid.

Reply:
Updated